packages/model-viewer/src/three-components/TextureUtils.ts

```diff
@@ -222,29 +219,35 @@ export default class TextureUtils {
     await timePasses();
 
     const renderer = this.threeRenderer;
-    const cubeTarget = new WebGLCubeRenderTarget(256, {
+    const tmpTarget = new WebGLCubeRenderTarget(256, {
       generateMipmaps: false,
       type: HalfFloatType,
       format: RGBAFormat,
       colorSpace: LinearSRGBColorSpace,
       depthBuffer: true
     });
-    const cubeCamera = new CubeCamera(0.1, 100, cubeTarget);
-    const generatedEnvironmentMap = cubeCamera.renderTarget.texture;
-    generatedEnvironmentMap.name = name;
+    const cubeCamera = new CubeCamera(0.1, 100, tmpTarget);
 
     const outputColorSpace = renderer.outputColorSpace;
     const toneMapping = renderer.toneMapping;
     renderer.toneMapping = NoToneMapping;
     renderer.outputColorSpace = LinearSRGBColorSpace;
 
-    cubeCamera.update(renderer, scene);
+    cubeCamera.update(renderer as unknown as WebGLRenderer, scene);
 
-    this.blurCubemap(cubeTarget, GENERATED_SIGMA);
+    const cubeTarget = new CubeRenderTarget(256);
+    (cubeTarget as any)
+        .fromCubeTexture(
+            this.threeRenderer, tmpTarget.texture, GENERATED_SIGMA);
```

Comment:
Here I'm relying on a WIP three.js PR: https://github.com/mrdoob/three.js/pull/30165. There's still some problem, probably in that PR, because randomly the generated environment either throws or creates a black texture.